course_catalog/constants.py

```diff
@@ -3,6 +3,10 @@
 """
 from enum import Enum
 
+OCW_BASE_URL = "http://ocw.mit.edu/"
+MITX_BASE_URL = "https://www.edx.org/course/"
+MITX_ALT_URL = "https://courses.edx.org/courses/"
```

Comment:
Might be a good idea to move these to `settings.py` as env vars with these as defaults just in case.